src/Microsoft.Fx.Portability/Microsoft.Fx.Portability.NET45.csproj

```diff
@@ -17,14 +17,14 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <DefineConstants>TRACE;DEBUG;NET45;FEATURE_ASSEMBLY_LOCATION;FEATURE_XML_SCHEMA</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NET45;FEATURE_ASSEMBLY_LOCATION;FEATURE_XML_SCHEMA;FEATURE_DEBUG_TRACE</DefineConstants>
```

Comment:
Hmmm... they may be now. I had disabled them when we first moved it over and they weren't available. They probably are now